src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -158,13 +159,15 @@ public void getElementsCountGroupByBoundaryGroupByTagTest() {
         + "/elements/count/groupBy/boundary/groupBy/tag?bboxes=8.68086,49.39948,8.69401,49.40609&"
         + "types=way&time=2016-11-09&keys=building&groupByKey=building&groupByValues=yes",
         JsonNode.class);
-    assertEquals(43, StreamSupport
-        .stream(Spliterators.spliteratorUnknownSize(
-            response.getBody().get("groupByResult").iterator(), Spliterator.ORDERED), false)
-        .filter(
-            jsonNode -> "boundary1".equalsIgnoreCase(jsonNode.get("groupByObject").get(0).asText())
-                && "remainder".equalsIgnoreCase(jsonNode.get("groupByObject").get(1).asText()))
-        .findFirst().get().get("result").get(0).get("value").asInt(), 0);
+    assertEquals(43,
+        StreamSupport
```

Comment:
I have no answer for that, it is automatically formatted. I'm not using specific format settings which regard this issue.

Reply:
ok, I see. If you ask me, this is not a good situation to use automatic code formatting, for multiple reasons:

* First, the code was already correctly formatted, so why change it?
* Since we're not all using the same IDEs, if another person uses a different auto code formatter, the same line would potentially be changed again. This could result in an "edit war" of the auto formatters.
* Such changes make it hard to read and review a pull request, because more lines are touched and it is sometimes hard to see whether something really changed or if it was "just" the reordering of a few newlines and whitespaces.
* Most importantly, any such line which is touched without a reason can lead to unnecessary merge conflicts with other branches. This can cause lots of manual work down the line.

You might choose to use an auto-formatter during your own development however you like, but if you do, please make sure not to commit unnecessary changes which were just introduced automatically. [`git add -p` is your friend](https://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/).

Reply:
I also ran into that issue in the past (and sometimes sitll today), which can indeed be a bit annoying and missleading.

What I try to remember myself always now is just marking the specific code lines, or method, that I've added/edited and then apply the auto formatting on that, instead of the whole class.
Sometimes it really changes something on a completely other part of the code, which also fits to the coding guidelines, but is just another way of ordering the lines..